ml-data-hub-plugin/src/main/groovy/com/marklogic/hub/gradle/task/FixCreatedByStepTask.groovy

```diff
@@ -28,6 +28,7 @@ class FixCreatedByStepTask extends HubTask {
         }
 
         println "Fixing datahubCreatedByStep metadata in database: " + database
+        println "Only documents with the necessary provenance data to determine the correct step name will be fixed"
```

Comment:
In other words hubFixCreatedByStep wont work for ingested records since they dont have the necessary prov?